modules/installation-launching-installer.adoc

```diff
@@ -227,6 +227,18 @@ The AWS access key ID and secret access key are stored in `~/.aws/credentials` i
 endif::aws[]
 ifdef::azure[]
 .. Select *azure* as the platform to target.
+.. You can store the credentials of the service principal in `~/.azure/osServicePrincipal.json`. In case that you use more 
+then one service principal can you define the environement variable `AZURE_AUTH_LOCATION` with the full path of the service principal json.
++
+[source,json]
+----
+{
+  "subscriptionId":"azure subscription id",
+  "tenantId":"azure tenant id",
+  "clientId":"azure service principal client id",
+  "clientSecret":"azure service principal client secret"
+}
+----
```

Comment:
This feels like more of a note to me than something that should be added as a step. It does not correlate with the prompt questions, so it feels out-of-place.

What are your thoughts on adding the info on the `AZURE_AUTH_LOCATION` as a note?

```
[NOTE]
====
If you regularly use more than one Azure service principal, you can define the environment variable `AZURE_AUTH_LOCATION` with the full path of the service principal JSON to use. For example, you can run the following command:
+
[source,terminal]
----
$ AZURE_AUTH_LOCATION=~/.azure/osServicePrincipal.json
----
====
```
I'm not sure if giving the JSON source of a service principal is necessary, since that's something an Azure user should be accustomed to seeing, especially if they're managing multiple service principals.